src/components/TestRunList/index.tsx

```diff
@@ -131,41 +119,46 @@ const TestRunList: React.FunctionComponent = () => {
   React.useEffect(() => {
```

Comment:
This ùseEffect` seems to be creating a forever update loop which needs to be solved